scripts/core/editor3/reducers/editor3.jsx

```diff
@@ -165,15 +182,19 @@ const changeImageCaption = (state, {entityKey, newCaption, field}) => {
     const {editorState} = state;
     const contentState = editorState.getCurrentContent();
     const entity = contentState.getEntity(entityKey);
-    let {img} = entity.getData();
+    let {media} = entity.getData();
+    let entityDataHasChanged = true;
```

Comment:
Please can you put this after line 194, exactly after the entity data changes. It makes more sense that way.

Reply:
done